### PR TITLE
Brian_L fun fact

### PR DESCRIPTION
**Pull request by tluu5**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,4 +130,4 @@ Clone this repository (or open in codespace), create a branch with your name, ad
 
 ## 2024-9.1
 | Name                 | Fun Fact |
-| -------------------- | --- |
+| Brian Luu            | If your DNA is off by 1 percentage point, you would be a dolphin. |
```
Comment (hatcheta1) [inline README.md:133]: Nice

**Review by hatcheta1 — APPROVED** [review]
Looks good!

**Review by AnnaVi11arrea1 — APPROVED** [review]
Excellent. probably could have left the blank line above to match the formatting of the previous cohorts.

**Review by AnnaVi11arrea1 — APPROVED** [review]
perfect

**Pull request merged**